docs/graphql/manual/mutations/upsert.rst

```diff
@@ -167,3 +167,18 @@ You can specify ``on_conflict`` clause while inserting nested objects
   To allow upserting in these cases, set ``update_columns: [<conflict-column>]``. By doing this, in case of a
   conflict, the conflicted column will be updated with the new value (which is the same value it had before and hence
   will effectively leave it unchanged) and will allow the upsert to go through.
+
+Upsert is not a substitute for update
```

Comment:
I would move this section as a child of the `convert insert mutation to upsert` _(and make content tree depth to 2)_ or just move it as a top level section right after that 

In its current location, as it is after the examples, I do not think it will have the desired impact as I am not sure if enough ppl will end up discovering it.